src/test-explorer/test-run-handler.ts

```diff
@@ -77,6 +77,7 @@ export async function runHandler(
 
         await analyzeResults(run, children, callback);
       } catch (error) {
+        callback();
```

Comment:
No callback seems to be sent after  `if (!wasStarted) {` also?

Reply:
God catch. I added `finally` callback